### ISSUE TITLE
这个网站的cookie爆破

### ISSUE THREAD

**Issue by wc110302** [SergioJune/Spider-Crack-JS#2]
url：https://www.vietjetair.com/Sites/Web/zh-CN/Home  这个网站进去后 点登陆-->机票代理 会跳转到登陆的界面去 这里面涉及到好几个混淆严重的js加密 大佬有空的话可以看看 登陆倒是不难 就是拿到这个登陆界面的静态html难

**Comment by SergioJune** [comment]
就是只需要拿到登陆页面的html即可？

**Comment by wc110302** [comment]
> 就是只需要拿到登陆页面的html即可？

是的。从首页跳转进登录页面拿到html，之前分析了很久好像都没啥结果，模拟登录那个步骤很简单就是一个post，但是拿到这个html有点困难

**Comment by jinbiao2** [comment]
作者解决登录页的问题了吗
